src/layman/layer/filesystem/input_file.py

```diff
@@ -406,7 +406,7 @@ def check_filenames(workspace, layername, input_files, check_crs, *, ignore_exis
                 assert old_match.group(0) == new_match.group(0)
 
     if not ignore_existing_files:
-        conflict_paths = [raw_filename_mapping[k]
+        conflict_paths = [v
                           for k, v in raw_filename_mapping.items()
```

Comment:
`for k, v in raw_filename_mapping.items()` -> `for v in raw_filename_mapping.values()`